JsonBuilder/JsonBuilder/JsonBuilder.cs

```diff
@@ -0,0 +1,225 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Dynamic;
+using System.Linq;
+using System.Text;
+using System.Web.Script.Serialization;
+
+namespace SettingSaver
+{
+    public enum Formatting
+    {
+        Indented,
+        Minify
+    }
+
+    public sealed class JsonBuilder
+    {
+        private readonly ExpandoObject _currentObject;
+        private static readonly JavaScriptSerializer JavaScriptSerializer = new JavaScriptSerializer();
+
+        /// <summary>
+        /// Create New Json Object
+        /// </summary>
+        /// <returns></returns>
+        public JsonBuilder()
+        {
+            this._currentObject = new ExpandoObject();
+        }
+
+        /// <summary>
+        /// Add new property to json object
+        /// </summary>
+        /// <param name="property">property Name</param>
+        /// <param name="value">property value</param>
+        /// <returns></returns>
+        public JsonBuilder AppendProperty(string property, dynamic value)
+        {
+            if (property == null)
```

Comment:
```csharp
if(String.IsNullOrEmpty(property))
    throw ArgumentNullException("The JSON properties can't be null!");
```

Reply:
But what if the user put null to the JSON property!?

Reply:
Or Enter a space that's not valid right?

Reply:
Ah, sorry got it!
Done